src/quo/components/calendar/calendar/component_spec.cljs

```diff
@@ -4,7 +4,7 @@
     [quo.components.calendar.calendar.view :as calendar]
     [test-helpers.component :as h]))
 
-(def start-date (time/date-time (time/year (time/now)) (time/month (time/now)) 5))
+(def start-date (time/date-time (time/year (time/now)) (time/month (time/now)) 8))
```

Comment:
We seem to display a calendar grid of 35 days, and this change attempts to avoid the calendar view displaying multiple day entries with the text "5". So by increasing the start-date to "8", it becomes unlikely that we'll find a duplicate entry for the 8th day of the next month. For example, the shortest month should 28 days (February) and that would only display the 7th day.

Reply:
Smart idea 👍🏼 

I was wondering if we shouldn't use `query-all-by-text` and getting the first one (if not empty)? The `query-all-*` versions don't throw when more than one element is found like `query-by-text`.

Reply:
Yeah that's probably a more future-proof way, I've added a commit to do that too 👍